docker/proxy.Dockerfile

```diff
@@ -24,6 +24,6 @@ FROM --platform=${TARGETPLATFORM:-linux/amd64} gcr.io/distroless/static:nonroot
 WORKDIR /
 COPY --from=builder /workspace/proxy .
 # Kubernetes runAsNonRoot requires USER to be numeric
-USER 65532:65532
+USER 1501:1501
```

Comment:
Can we keep `USER` the same or uid `1501` is a special user that we should use here?

Reply:
`65532` is commonly defined in Dockerfile to bypass the security scanners. We want to pick a UID that will most likely not conflict with other users running in the pod, so the redirection works as expected. It doesn't necessarily have to be `1501`. I just picked a random one to avoid conflicts with other sidecar proxies (service meshes). Lmk what you think!

Reply:
Makes sense